frontend/src/concepts/pipelines/content/createRun/submitUtils.ts

```diff
@@ -69,6 +68,24 @@ const convertDateDataToKFDateTime = (dateData?: RunDateTime): DateTimeKF | null
   return date.toISOString();
 };
 
+export const convertPeriodicTimeToSeconds = (timeString: string): number => {
+  const numericValue = parseInt(timeString, 10);
```

Comment:
This can also return `NaN` if the timeString is something like `"hour12"`

```node
> parseInt("hour12", 10)
NaN

> NaN * 60 * 60;
NaN
```

If this is expected, also include this output in tests.